src/components/Pama/pama.jsx

```diff
@@ -68,6 +68,27 @@ const dispatchRatingUpdates = (dispatch, updates) => updates
     rating,
   }));
 
+const dispatchResourceUpdate = (dispatch, update) => {
+  if (update.resource.code && update.resource.code.coding.length > 0) {
+    dispatch({
```

Comment:
Quick question: will these two `dispatch()` invocations result in two distinct service exchange rounds? If so, might be good to combine them in an atomic `types.SET_STUDY_AND_REASON` aciton, or some-such.

Reply:
Correct it would dispatch two service exchanges.  Just committed a refactoring to create a new action type.